dist/tools/nacl_key_generator/generate-nacl-keys.c

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
>which runs local tools 

local, but tested and proven solid tools. Such tools might also be more portable than a self-made one (is there a `/dev/random` for OSX/Windows)?